src/main/java/com/onelogin/saml/Utils.java

```diff
@@ -53,10 +75,12 @@ public String getNamespaceURI(String prefix) {
 				String result = null;
 				if (prefix.equals("samlp"))
```

Comment:
ok, but which constant should be associated? on php toolkit Utils I only see saml and samlp